### PR TITLE
Feature/grab bag of org stuff

### PR DESCRIPTION
**Pull request by agduncan94**
Lots of small changes
* Changed Available Organizations title to Organizations
* Swap events and description tabs for organizations
* Only let members/maintainers edit readme
* Only show members who have accepted
* Message for how to add entries to a collection
* Alert bar if org/collection is pending
* Properly close add entry to collection modal
* Fix spacing on add entry to collection modal buttons
* Can only edit memberships if you are a maintainer
* Fix event spacing
* Improved create org request hints
* Better display of collections and orgs on mobile
* Cleanup of members tab
* and more small changes

pendingorg

pendingorgmembers

### CODE REVIEW

**Comment by denis-yuen** [comment]
Ah merge conflict

File: src/app/organizations/collection/collection.component.html [hunk 1]
```diff
@@ -44,15 +48,15 @@ <h1><a [routerLink]="['/organizations/', organization.id]">{{ organization.name
       </mat-card>
 
       <mat-card class="alert alert-info" *ngIf="collection?.entries?.length == 0">
-        This collection has no associated entries
+        This collection has no associated entries. To add an entry, navigate to the public page of the tool or workflow you want to add and select `add to collection`
```
Comment (coverbeck) [inline src/app/organizations/collection/collection.component.html]: Should ``add to collection`` be cased properly to match the case when displayed on the button? Or is this a recommendation for how cards are styled?

File: src/app/organizations/collection/collection.component.html [hunk 2]
```diff
@@ -21,6 +21,10 @@
 <app-loading [loading]="(loadingCollection$ | async) && (loadingOrganization$ | async)">
   <div *ngIf="(organization$ | async) as organization">
     <div class="container" *ngIf="(collection$ | async) as collection" fxLayout="column">
+      <mat-card fxFlex class="my-3 alert alert-info" *ngIf="organization?.status === 'PENDING'">
```
Comment (garyluu) [inline src/app/organizations/collection/collection.component.html]: Enum

File: src/app/organizations/collection/collection.component.html [hunk 3]
```diff
@@ -63,12 +67,8 @@ <h4>
               </mat-card-subtitle>
             </mat-card-header>
 
-            <mat-card-content class="p-3">
-              <mat-chip-list>
-                <mat-chip *ngFor="let label of entry?.labels">{{ label.value }}</mat-chip>
-              </mat-chip-list>
-            </mat-card-content>
             <mat-card-actions *ngIf="(canEdit$ | async)">
+              <div fxFlex></div>
```
Comment (garyluu) [inline src/app/organizations/collection/collection.component.html]: Not what it's intended for, creates a 0 height element.  Meant to actually have content inside it.  Just do the same thing as add-entry.component.html

File: src/app/organizations/organization-members/organization-members.component.html [hunk 4]
```diff
@@ -14,30 +14,30 @@
    limitations under the License.
 -->
 <div>
-  <app-loading [loading]="(loading$ | async)">
-    <div fxLayout="row wrap" fxLayoutAlign="start stretch">
-      <div *ngFor="let organizationUser of (organizationMembers$ | async)">
-        <!-- Fixed width 200px is bad for long usernames -->
-        <mat-card class="my-2" fxFlex="200px" fxFlexOffset="10px">
+    <app-loading [loading]="(loading$ | async)">
+    <div class="pt-2"></div>
```
Comment (garyluu) [inline src/app/organizations/organization-members/organization-members.component.html]: try sticking the class in the div below

File: src/app/organizations/collection/collection.component.html [hunk 5]
```diff
@@ -41,15 +45,15 @@ <h1><a [routerLink]="['/organizations/', organization.id]">{{ organization.name
...
 
-      <div fxLayoutGap="16px grid" fxLayout="row wrap">
-        <div *ngFor="let entry of collection?.entries" fxFlex="{{ 100/3 }}%">
+      <div fxLayoutGap="16px grid" fxLayout="row wrap" fxLayout.lt-sm="column">
+        <div *ngFor="let entry of collection?.entries" fxFlex="50%" fxFlex.lt-sm="100%">
           <mat-card class="entryCard">
             <mat-card-header>
               <mat-card-title>
-                <h4>
+                <h4 style="overflow-x: wrap">
```
Comment (garyluu) [inline src/app/organizations/collection/collection.component.html]: This likely doesn't do anything.

File: src/app/organizations/collection/add-entry/add-entry.component.html [hunk 6]
```diff
@@ -33,10 +33,10 @@ <h1 mat-dialog-title>Add to collection</h1>
     </div>
   </app-loading>
 </div>
-<div mat-dialog-actions align="end">
+
+<div mat-dialog-actions fxLayout="row" fxLayoutAlign="flex-end center" fxLayoutGap="10px">
```
Comment (garyluu) [inline src/app/organizations/collection/add-entry/add-entry.component.html]: https://github.com/angular/flex-layout/wiki/Declarative-API-Overview just use 'end' like the doc says, my editor doesn't even recognize flex-end as valid

**Pull request merged**